rmw_fastrtps_cpp/src/rmw_count.cpp

```diff
@@ -51,8 +51,8 @@ rmw_count_publishers(
   WriterInfo * slave_target = impl->secondaryPubListener;
   slave_target->mapmutex.lock();
   *count = 0;
-  for (auto it : slave_target->topicNtypes) {
-    auto topic_fqdn = _demangle_if_ros_topic(it.first);
+  for (const auto & it : slave_target->topicNtypes) {
+    auto const topic_fqdn = _demangle_if_ros_topic(it.first);
```

Comment:
Please use `const auto` / `auto const` consistently throughout the code base.

Reply:
I started the other way around and missed updating these two.
Fixed now